lib/presentation/screen/main_screen.dart

```diff
@@ -40,45 +40,34 @@ class _MainScreenState extends State<MainScreen> {
   @override
   Widget build(BuildContext context) {
     return GradientScaffold(
```

Comment:
Remove Gradient scaffold from the tabs widget, (Home Screen and Profile Screen). simply use transparent scaffold for appbar.